tools/serverpod_cli/lib/src/create/create.dart

```diff
@@ -35,6 +38,10 @@ Future<bool> performCreate(
   ServerpodTemplateType template,
   bool force,
 ) async {
+  if (name == '.') {
```

Comment:
Ok, but I feel like we should at least put a note here as it is a bit of a hack. It would be best to support `serverpod create mypod` format to perform upgrades also.